File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/21/21.go
// Q21: Print palindromic number triangle (1, 21, 321, 4321).
// Input: An integer n
// Output: Decreasing number triangle

package main

import "fmt"

func main() {
    var n int
    fmt.Scan(&n)
    for i := 1; i <= n; i++ {
        row := ""
        for j := i; j >= 1; j-- {
            row += fmt.Sprintf("%d", j)
        }
        fmt.Println(row)
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/21/21.py
# Q21: Print palindromic number triangle (1, 21, 321, 4321).
# Input: An integer n
# Output: Decreasing number triangle

n = int(input())
for i in range(1, n + 1):
    for j in range(i, 0, -1):
        print(j, end="")
    print()
